nautobot/extras/urls.py

```diff
@@ -482,6 +486,20 @@
         views.JobResultDeleteView.as_view(),
         name="jobresult_delete",
     ),
+    # Job Button
+    path(
+        "job-button/<uuid:pk>/changelog/",
+        views.ObjectChangeLogView.as_view(),
+        name="jobbutton_changelog",
+        kwargs={"model": JobButton},
+    ),
+    path(
+        "job-button/<uuid:pk>/notes/",
+        views.ObjectNotesView.as_view(),
+        name="jobbutton_notes",
+        kwargs={"model": JobButton},
+    ),
```

Comment:
These should be included with a `NautobotUIViewSet`, `NautobotUIViewSetRouter` now: https://github.com/nautobot/nautobot/pull/2752, https://docs.nautobot.com/projects/core/en/stable/plugins/development/#nautobotuiviewsetrouter